rest-client/pom.xml

```diff
@@ -33,6 +33,16 @@
 			<groupId>io.vertx</groupId>
 			<artifactId>vertx-rx-java2</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>okhttp</artifactId>
+			<version>3.12.1</version>
```

Comment:
Not a must have but I tend to use properties for central dependency versions. That makes it easier to update the whole stack.